Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -920,7 +920,6 @@ def sanitize_ARGV_and_ENV
     end
 
     ENV["HOMEBREW_DEVELOPER"] = "1"
-    ENV["HOMEBREW_NO_AUTO_UPDATE"] = "1"
```

Comment:
See https://github.com/Homebrew/brew/pull/669/commits/3c901a972fc57d94763c86965f23e9796a9504c5. It's moved to `brew.sh` instead which means it doesn't need duplicated around every command that needs it.